Scripts/GameManager.gd

```diff
@@ -24,11 +25,24 @@ func _ready() -> void:
 	self.debug = false
 
 func players_spawned() -> void:
+	current_players = players.duplicate()
 	emit_signal("players_spawned")
 
-func game_over(player_killed: String) -> void:
+func kill_player(player_killed: String) -> void:
+	if len(current_players) <= 1:
+		return
+	
+	for i in len(current_players):
```

Comment:
I would say to use `[].size()` because it is a native method of arrays, while `len()` is a global GDScript function:

```suggestion
	if current_players.size() <= 1:
		return
	
	for i in current_players.size():
```

Reply:
I think that we had decided to stick with `len()` for consistency 😅 
Should we start using `size()` and update that in other scripts as well?

Reply:
Humm, ye, but I think we should do that in other day, probably create an issue for that 🤷‍♂️

(I know that in python `size()`/`count()`/`length()`/etc... doesn't exist and we need to use `len()`, thanks GDScript 🙏😂)